### DISCUSSION TITLE
无法启动milvus-etcd的镜像，failed to start milvus-etcd

### DISCUSSION THREAD

**Discussion by zlw18813081566** [milvus-io/milvus#25057] [Q&A and General discussion]
panic: freepages: failed to get all reachable pages (page 12961: multiple references)

![图片](https://github.com/milvus-io/milvus/assets/84000900/fe46a68d-bbaf-4353-9e51-42363512ee03)

**Comment by yhmo** [comment]
参考这个： https://zhuanlan.zhihu.com/p/558940973
